src/Containers/Mount/Bind.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Utopia\Containers\Mount;
+
+use Utopia\Containers\Mount;
+
+final readonly class Bind implements Mount {
+    public function __construct(
+        public string $hostPath,
+        public string $containerPath,
+        public bool $readOnly = false
```

Comment:
I would mark them `protected` instead, so you cannot directly read, or change them. And must use `get` or `set` methods.